### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by xLuge** [tsiendragon/qwen-image-finetune#27]
The training data should be normalized to -1~1?

**Comment by xLuge** [comment]
After run_validation() self.scheduler.timesteps is change. Training could not continue.

**Comment by tsiendragon** [comment]
```
logging:
  output_dir: ./image_edit_demo/  # change the path to your own output path
  report_to: tensorboard
  # report_to: wandb
  # report_to: swanlab
  tracker_project_name: DemoFluxKontextFp16
  tags:
    - test
    - FLuxKontext
    - FaceSeg
    - lilong
  notes: "This is a test configuration for FluxKontext on FaceSeg dataset"
  sampling:  
    enable: True
    validation_steps: 100
    num_samples: 1
    seed: 42
```
Do not add sampling in logging
Refer the configs in `tests/test_configs/test_example_fluxkontext_fp16.yaml`

**Comment by xLuge** [comment]
> ```
> logging:
>   output_dir: ./image_edit_demo/  # change the path to your own output path
>   report_to: tensorboard
>   # report_to: wandb
>   # report_to: swanlab
>   tracker_project_name: DemoFluxKontextFp16
>   tags:
>     - test
>     - FLuxKontext
>     - FaceSeg
>     - lilong
>   notes: "This is a test configuration for FluxKontext on FaceSeg dataset"
>   sampling:  
>     enable: True
>     validation_steps: 100
>     num_samples: 1
>     seed: 42
> ```
> 
> Do not add sampling in logging Refer the configs in `tests/test_configs/test_example_fluxkontext_fp16.yaml`

This doesn’t seem to be important. It’s missing the step to convert "img_shapes" from pixel space to latent shapce。

**Comment by xLuge** [comment]
> ```
> logging:
>   output_dir: ./image_edit_demo/  # change the path to your own output path
>   report_to: tensorboard
>   # report_to: wandb
>   # report_to: swanlab
>   tracker_project_name: DemoFluxKontextFp16
>   tags:
>     - test
>     - FLuxKontext
>     - FaceSeg
>     - lilong
>   notes: "This is a test configuration for FluxKontext on FaceSeg dataset"
>   sampling:  
>     enable: True
>     validation_steps: 100
>     num_samples: 1
>     seed: 42
> ```
> 
> Do not add sampling in logging Refer the configs in `tests/test_configs/test_example_fluxkontext_fp16.yaml`

run with pre cache is ok! nice bro!
